src/rev_ai/apiclient.py

```diff
@@ -322,8 +341,29 @@ def get_account(self):
 
         :raises: HTTPError
         """
-        url = urljoin(self.base_url, 'account')
-        response = self.session.get(url)
-        response.raise_for_status()
+        response = self._HTTPHandler(
+                      "GET",
+                      urljoin(self.base_url, 'account')
+        )
 
         return Account.from_json(response.json())
+
+    def _HTTPHandler(self, method, url, **kwargs):
+        """Wrapper method for initiating HTTP requests and handling potential
```

Comment:
I like that you documented this. But just for future reference, since it's a private method it doesn't really NEED to be documented